src/tokio.rs

```diff
@@ -318,7 +325,7 @@ pub fn future_into_py_with_locals<F, T>(
 ) -> PyResult<Bound<PyAny>>
 where
     F: Future<Output = PyResult<T>> + Send + 'static,
-    T: for<'py> IntoPyObject<'py>,
+    T: for<'py> IntoPyObject<'py> + Send + 'static,
```

Comment:
This is a breaking change, right? Should we highlight that in the Changelog? What if someone wants to use this with a non-`Send` return type?